apps/rush-lib/src/cli/actions/InstallAction.ts

```diff
@@ -7,6 +7,8 @@ import { RushCommandLineParser } from '../RushCommandLineParser';
 import { SelectionParameterSet } from '../SelectionParameterSet';
 
 export class InstallAction extends BaseInstallAction {
+  // must match name of _selectionParameters in BaseInstallAction for telemetry to work
+  // worthy of the override parameter in TypeScript > 4.3
   protected _selectionParameters!: SelectionParameterSet;
```

Comment:
Why does this need to be redeclared?

Reply:
Is it because this one is required and the base one isn't?

Reply:
The reality is that it only needs to be initialized which that line didn't enforce so I removed this declaration, which would at least cause an error if the name were to change